pkg/pquerier/pqueriertest/dataframe_query_integration_test.go

```diff
@@ -856,3 +856,106 @@ func (suite *testSelectDataframeSuite) TestQueryNonExistingMetric() {
 		}
 	}
 }
+
+func (suite *testSelectDataframeSuite) TestSparseStringAndNumericColumnsDataframe() {
```

Comment:
Please also add tests to other cases, such as
1. where you have 2 labels and one metric is not present in one of the labels
2. where you request for a not existing metric
3. if you can think of something else..